.coveragerc

```diff
@@ -1,4 +1,5 @@
 [report]
 exclude_lines =
-    pragma: nocover
-    if TYPE_CHECKING:
+    pragma: no cover
+    if TYPE_CHECKING
+    ^\s\.\.\.\s$
```

Comment:
Ellipsis can exist without `@overload` decorator, so why just not ignore ellipsis?

Reply:
Indeed it can, but the PR's description indicated that it's ignored specifically to ignore type overloads, for which I think a more explicit `@overload` would make more sense.

Elipsis can indeed be used in other places too, it's not unique to just overloads (for example it's commonly seen in abstractmethods). So the question is whether it really makes sense to ignore coverage for all possible use-cases of it. Since I currently can't think of any valid use-cases with elipsis where coverage shouldn't be ignored, it probably can stay.

Reply:
So conversation resolved?

Reply:
(Actually in abstractmethods you should use docstrings, they will work just like elipsis do.)

Reply:
> So conversation resolved?

Unless someone else can think of a use-case which I can't, yes